packages/app/src/server/models/config.ts

```diff
@@ -245,6 +245,8 @@ schema.statics.getLocalconfig = function(crowi) {
     globalLang: crowi.configManager.getConfig('crowi', 'app:globalLang'),
     pageLimitationL: crowi.configManager.getConfig('crowi', 'customize:showPageLimitationL'),
     pageLimitationXL: crowi.configManager.getConfig('crowi', 'customize:showPageLimitationXL'),
+    isDefaultLogo:  crowi.configManager.getConfig('crowi', 'customize:isDefaultLogo'),
+    uploadedLogoSrc: crowi.configManager.getConfig('crowi', 'customize:uploadedLogoSrc'),
```

Comment:
- クライアント向けには isDefaultLogo 不要
- uploadedLogoSrc を customizedLogoSrc として提供し、isDefaultLogo: true なら値を入れない

Reply:
- No isDefaultLogo required for clients.
- Provide uploadedLogoSrc as customizedLogoSrc and do not assign value if isDefaultLogo: true

Reply:
Did you read this?

Reply:
`isDefault` Logo and `customizedLogoSrc` are returned to config.ts because they are both used in GrowiNavbar to handle use of `default logo` or `upload logo` with logo or without logo uploaded

Reply:
Yuki said that all what's defined here are for Admin page and NOT explicitely for Customize page.

It is sure that you need `isDefaultLogo` to handle use of default logo or upload logo.
But `isDefaultLogo` should not be defined here for the Admin page.

Reply:
There is a condition when the page is accessed by the client, `isDefaultLogo` is also used by the `GrowiNavbar` component, so it will be fetched from `localConfig or appContainer.config`. However `customizedLogoSrc` and `isDefaultLogo` are still used to specify the image source in "Upload Logo". Please suggest

Reply:
```
    const customizedLogoSrc = isHoge
        ? valueA
        : valueB
```